packages/arb-token-bridge-ui/tests/e2e/README.md

```diff
@@ -0,0 +1,19 @@
+# End-to-end tests
```

Comment:
Sweet! 

Let's link to this README from the main README, and also add documentation around copying the sample .env file.